src/app/services/[id]/edit/page.tsx

```
@@ -26,6 +27,10 @@ const EditServicePage = async ({
   const service = await getService(db, params.id);
   if (!service) notFound();
 
+  const providerVariables = service.providers
+    .map(provider => getProviderVariables(service.name, provider.name, provider.variables))
+    .reduce((bundle, current) => ({ ...bundle, ...current }), {});
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)